src/Samples/FileDialogs.Core/Program.fs

```diff
@@ -1,9 +1,7 @@
-﻿module Elmish.WPF.Samples.FileDialogs.Program
+module Elmish.WPF.Samples.FileDialogs.Program
```

Comment:
I am very confused about why GH sometimes show these diffs on the first line. Do you know why? I would guess the file has been saved for the first time in UTF-8 with a BOM (Byte Order Mark) at the start; could that be it?

You don't have to revert it, I'm just curious.

Reply:
I had to look at a HEX editor to see the difference.  It sounds like your guess is correct.  I don't know what those bytes mean though.

## File in `master`
![2021-02-19_22-20-54_524_chrome](https://user-images.githubusercontent.com/34664007/108583712-2fdb6100-7301-11eb-95ca-a3df5ab56ec0.png)

## File in PR's branch
![2021-02-19_22-21-00_525_chrome](https://user-images.githubusercontent.com/34664007/108583719-3a95f600-7301-11eb-8ae6-cb8a8e8f6fc5.png)

Reply:
https://en.wikipedia.org/wiki/Byte_order_mark